src/containers/Application/ApplicationPageWrapper.tsx

```diff
@@ -183,8 +206,10 @@ const ApplicationPageWrapper: React.FC = () => {
       })
     })
 
-    if (!revalidate.allValid) setShowModal({ open: true, ...messages.VALIDATION_FAIL })
-    else push(`/application/${serialNumber}/summary`)
+    if (!revalidate.allValid) {
+      setShowModal({ open: true, ...messages.VALIDATION_FAIL })
+      setSummaryButtonClicked(false)
+    } else push(`/application/${serialNumber}/summary`)
```

Comment:
Should do the same in the `else` statement?

Reply:
No need I think, since the Summary page (where it gets sent) has no interest in the state of Summary button. (And if you tried to back to this page, the state is reset.)